src/biotite/structure/io/pdbx/convert.py

```diff
@@ -1616,3 +1617,56 @@ def _convert_string_to_sequence(string, stype):
         raise InvalidFileError(
             "mmCIF _entity_poly.type unsupported" " type: " + stype
         )
+
+def get_sse(pdbx_file, data_block=None):
+    """
+    Gets secondary structure from pdbx file
+
+    Parameters
+    ----------
+    pdbx_file : CIFFile or CIFBlock or BinaryCIFFile or BinaryCIFBlock
+        The file object.
+    
+    Returns
+    ----------
+    sec_struct_dic: keys are the different chains from the pdbx file 
```

Comment:
Similar to the description of a parameter [Numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#returns) uses the form
```
some_name : some_type
    Some description.
```